konfera/event/views.py

```diff
@@ -127,3 +128,13 @@ def event_public_tickets(request, slug):
 
     context['tickets'] = available_tickets
     return render(request=request, template_name='konfera/event_public_tickets.html', context=context)
+
+
+def event_order_detail(request, slug, order_uuid):
+    context = dict()
+    event = get_object_or_404(Event.objects.published(), slug=slug)
```

Comment:
So orders can only have tickets from the same event? Because if an order can have tickets from multiple events, I thought of adding the event as a column in the related tickets table as in image below.

![screenshot 2016-10-26 12 08 27](https://cloud.githubusercontent.com/assets/3467229/19736608/6cfa5c1e-9b86-11e6-82da-c9d952a3be0d.png)

Another thing I was thinking is to put the label in the order status for better viewing, like in the previous and below images.

![screenshot 2016-10-26 14 16 56](https://cloud.githubusercontent.com/assets/3467229/19736807/3be7b396-9b87-11e6-8ee9-0065f94707d3.png)